packages/network_info_plus/network_info_plus/example/ios/Runner/Info.plist

```diff
@@ -24,6 +26,10 @@
 	<string>$(FLUTTER_BUILD_NUMBER)</string>
 	<key>LSRequiresIPhoneOS</key>
 	<true/>
+	<key>NSLocationWhenInUseUsageDescription</key>
+	<string>Obtain WIFI Name</string>
```

Comment:
Added the location permission to the info.plist as recommended in the plugin readme